libcudacxx/include/cuda/std/__functional/binary_negate.h

```diff
@@ -21,6 +21,7 @@
 #  pragma system_header
 #endif // no system header
 
+#include <cuda/std/__cccl/diagnostic.h>
```

Comment:
Those includes should not be necessary, all `__cccl` headers are directly included within the config header above